cms/admin/pageadmin.py

```diff
@@ -104,6 +104,7 @@ class PageAdmin(admin.ModelAdmin):
     copy_form = CopyPageForm
     move_form = MovePageForm
     inlines = PERMISSION_ADMIN_INLINES
+    search_fields = ('=id', 'urls__slug', 'pagecontent_set__title', 'reverse_id')
```

Comment:
I believe this `=id` is a typo?

Reply:
No that's OK.

That makes it an `iexact` search on ID

See examples in the docs
https://docs.djangoproject.com/en/5.1/ref/contrib/admin/#django.contrib.admin.ModelAdmin.search_fields

Reply:
Ah, right!

Reply:
I've translated the search fields from the page content admin for consistency. :-)